pkgs/development/python-modules/uranium/lulzbot.nix

```diff
@@ -0,0 +1,46 @@
+{ stdenv, fetchurl, fetchgit, buildPythonPackage, fetchFromGitHub, python, cmake
+, pyqt5, numpy, scipy, shapely, libarcusLulzbot, doxygen, gettext, pythonOlder }:
+
+buildPythonPackage rec {
+  version = "3.6.10";
+  pname = "uranium";
+  name = "uraniumLulzbot";
+  format = "other";
+
+  src = fetchgit {
+    url = https://code.alephobjects.com/diffusion/U/uranium.git;
+    rev = "33df88a7414375ac924ac761113baa48d2ced2b4";
+    sha256 = "109cbv7y105crbrzfp70lmcr9n20ap5c97i5qd46fmxbx86yj7f8";
+  };
+
+  disabled = pythonOlder "3.5.0";
+
+  buildInputs = [ python gettext ];
+  propagatedBuildInputs = [ pyqt5 numpy scipy shapely libarcusLulzbot ];
+  nativeBuildInputs = [ cmake doxygen ];
+
+  patches = [
+    # Qt 5.12+ support; see https://code.alephobjects.com/rU70b73ba0a270799b9eacf78e400aa8b8ab3fb2ee
+    (fetchurl {
+      url = "https://code.alephobjects.com/file/data/bwlg4bwkv4zzqaju4dyi/PHID-FILE-gmlyi636lqqw5ja2il3v/file?diff=1";
```

Comment:
This URL no longer works, I get:
```
trying https://code.alephobjects.com/file/data/bwlg4bwkv4zzqaju4dyi/PHID-FILE-gmlyi636lqqw5ja2il3v/file?diff=1
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (22) The requested URL returned error: 404 Not Found
error: cannot download file?diff=1 from any mirror
builder for '/nix/store/wyxvmn9rfw6whh81s75pdyw9w5y3f0zd-file?diff=1.drv' failed with exit code 1
```

Reply:
Well, that's disappointing. If I navigate to the same address now I get the URL https://code.alephobjects.com/file/data/l4x4gxrrqorjwh6twrr2/PHID-FILE-lgeooxasr6hwfqtdzvoo/file?diff=1 -- but my guess is that that one isn't going to be any more stable.

Might just fold it into the source tree. Thank you for checking!